src/patient-registration/patient-registration-validation.test.tsx

```diff
@@ -42,3 +42,42 @@ describe('telephone number input', () => {
   testInvalidPhoneNumber('+0800001066');
   testInvalidPhoneNumber('(0800)001066');
 });
+
+describe('contact person telephone number input', () => {
```

Comment:
Look good to me 🚀

Reply:
Think we should do that as part of the validation ticket. That involves quite some refactoring anyway. I feel it makes sense like this as part of the contact person context.

Reply:
Good point, I'll mark this as resolved